src/rabbit_recovery_terms.erl

```diff
@@ -204,7 +204,7 @@ open_table(VHost) ->
         file:delete(File),
         %% Sleep for a period of time to avoid the CPU surge caused by repeated operation
         Wait_time = 1000,
-        rabbit_log:warning("open '~p' failed, deleted it and try again after ~pms.", 
+        rabbit_log:warning("Failed to open '~p', deleted it and try again after ~pms.", 
```

Comment:
We would update the copywriting of log messages on your behalf. Don't worry about it.